### DISCUSSION TITLE
ERROR: Cache, or information obtained from the security directory is newer than repository

### DISCUSSION THREAD

**Discussion by pikdum** [borgbase/vorta#1681] [FAQ] [answered]
I'm getting this error when trying to backup today:

> ERROR: Cache, or information obtained from the security directory is newer than repository - this is either an attack or unsafe (multiple repos with the same ID)

I have Vorta installed as a Flatpak.

Any ideas on how to fix?

Thanks!

**Comment by pikdum** [comment, marked as answer]
Figured out what happened:

My NAS replaced the NFS share with an old ZFS clone, after a reboot.
No idea why it would do that, but that's what triggered this error.

**Comment by smartm0use** [comment]
I'm encountering the same error.

I have a repository to backup an external usb drive, and its path was something like `/media/<UUID-drive>/stuff`. Some time ago I applied a name to the backup drive so its path became `/media/<my-backup-drive-name>/stuff`. Today when I have tried to mount one of the archives it says that it can't find that path because it was pointing to the old path (the one containing the UUID).
So I have removed the repository and tried to add it again with the current path... but I get the error described above.

Could be the case to run the following commands as described [here](https://borgbackup.readthedocs.io/en/stable/faq.html#this-is-either-an-attack-or-unsafe-warning)?

```
borg config repo id   # shows the REPO_ID
rm ~/.config/borg/security/REPO_ID/manifest-timestamp
borg delete --cache-only REPO
```

I gave it a try but it doesn't fix the problem. What could I solve this issue?

**Reply by real-yfprojects** [reply]
Join the borg IRC on [libera chat](https://web.libera.chat/?nick=Guest&#borgbackup) or [matrix](https://matrix.to/#/#borgbackup:libera.chat).

**Reply by smartm0use** [reply]
I did, what's next now? :)

**Reply by capi** [reply]
You quit the channel after 14min, which might be a bit tight for people to answer. Consider different time-zones, people being at work, etc.

**Reply by m3nu** [reply]
I'd first run `borg info` on the moved repo so you can confirm any security questions. Vorta can't always do that, since it's non-interactive.

If that fails, I'd try moving cache and keys for this repo and start over:

` ~/.cache/borg/REPO_ID` and `~/.config/borg/security/REPO_ID` I think.

**Reply by smartm0use** [reply]
@capi I got disconnected and I have not noticed it. I think that joining via browser was not a good idea...

**Comment by smartm0use** [comment]
@m3nu Sorry I didn't get what I have to do with `borg info`...
About moving cache and keys... what is this for?

**Reply by m3nu** [reply]
It's to just run any Borg command and see if there is an error. Could be any other command.

**Reply by smartm0use** [reply]
`borg` works, in fact I run the above commands and it didn't return errors. But I still can't access to my backup :(

**Comment by smartm0use** [comment]
Ok, thanks to the hint of @m3nu I solved removing these folders:

```
<my-home>/.var/app/com.borgbase.Vorta/cache/borg/<repo-id>
<my-home>/.var/app/com.borgbase.Vorta/config/borg/security/<repo-id>
```

**Comment by taneharre** [comment]
Linux Mint 22
For me removing the ~/cache/borg/ and ~/.config/borg/security file got it working.

Caused by reinstalling OMV 7 and copying all its old files over.
